ydb/docs/en/core/concepts/glossary.md

```diff
@@ -272,6 +272,20 @@ A **tablet generation** is a number identifying the reincarnation of the tablet
 
 A **tablet local database** or **local database** is a set of data structures and related code that manages the tablet's state and the data it stores. Logically, the local database state is represented by a set of tables very similar to relational tables. Modification of the state of the local database is performed by local tablet transactions generated by the tablet's user actor.
 
+Each local database table is stored using the [LSM tree](#lsm-tree) data structure.
+
+#### LSM tree {#lsm-tree}
+
+An **LSM tree**, or [log-structured merge-tree](https://en.wikipedia.org/wiki/Log-structured_merge-tree), is a data structure designed to optimize write and read performance in storage systems and used in {{ ydb-short-name }} to store local database tables.
+
+#### MemTable {#memtable}
+
+All incoming data is initially written to an in-memory structure called a **MemTable**. When the MemTable reaches a predefined size, it is flushed to disk as an immutable [SST](#sst) file.
```

Comment:
```suggestion
All data written to a [tablet local database](#local-database) is initially stored in an in-memory data structure called a **MemTable**. When the MemTable reaches a predefined size, it is flushed to disk as an immutable [SST](#sst).
```
"All incoming data" needs clarification on where the data is getting into. It's definitely not YDB as a whole. I'm kind of guessing based on the rest of the content; please double-check.